lua/crates/toml.lua

```diff
@@ -774,10 +777,19 @@ function M.parse_crates(buf)
             dep_section = M.parse_section(section_text, line_nr, section_start - 1)
             dep_section_crate = nil
             if dep_section then
+                if dep_section.name then
+                    table.insert(working_crates, {
+                        name = dep_section.name,
+                        span = dep_section.name_col,
+                        kind = types.WorkingCrateKind.TABLE,
+                        line = i,
```

Comment:
this should be `line_nr` instead of `i`, the internal representation of all columns and lines is 0-based.